app/services/itemizable_update_service.rb

```diff
@@ -58,4 +60,21 @@ def self.update_storage_location(itemizable:, apply_change_method:, undo_change_
     # Apply the new changes to the storage location inventory
     to_location.public_send(apply_change_method, itemizable.line_item_values)
   end
+
+  # @param itemizable [Itemizable]
+  # @param from_location [StorageLocation]
+  # @param to_location [StorageLocation]
+  def self.verify_intervening_audit_on_storage_location_items(itemizable:, from_location_id:, to_location_id:)
+    return unless from_location_id != to_location_id && Audit.finalized_since?(itemizable, [from_location_id, to_location_id])
```

Comment:
Can we change this into an `if` and swap the conditions? I find `unless x && y` to be very confusing.